### PR TITLE
Add annimation to the map

### PR DESCRIPTION
**Pull request by annimation**
Adding Toronto to the map!

closes #9057

### CODE REVIEW

**Comment by iKev** [comment]
@annimation , you should remove the numeral (#githubHandle) to complete your pull request.
:simple_smile:

**Comment by dravesb** [comment]
Make sure you put the keyword closes. Nice work @annimation! 🔥

**Comment by Mij-Strebor** [comment]
Ok. I'm not understaning why the travis-cl/pr build failed is on this pull request message is showing. Who's pulled what, doing what building, what are they building, and why do I, as a colab, care?  If travis plays, have fun.

File: _pins/annimation.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+#githubHandle:annimation
```
Comment (crichID) [inline _pins/annimation.json]: @annimation you will need to remove the `#` at the beginning of this line and add a space after each `:` in order for the tests to pass. If you need any help, please check out this video: https://vimeo.com/201011288

**Comment by crichID** [comment]
>  I'm not understaning why the travis-cl/pr build failed is on this pull request message is showing. Who's pulled what, doing what building, what are they building, and why do I, as a colab, care?

@Mij-Strebor in this repository, Travis is mostly running a yaml linter. So the majority of the failures are related to incorrect syntax in the file being created. 

The map with all of the pins (shown on the first page of the course) is generated using a Jekyll collection. We take all of the values in the yaml front matter and combine it into a single topojson. If someone merges a file that contains incorrect syntax, it will cause this collection to fail. This is why we test for the correct syntax in all of the pins before they can be merged. Does that help?

**Pull request merged**